src/App.js

```diff
@@ -3,34 +3,51 @@ import withFirebaseAuth from 'react-with-firebase-auth'
 import * as firebase from 'firebase/app';
 import 'firebase/auth';
 import firebaseConfig from './firebaseConfig';
-import logo from './logo.svg';
+import logo from './icon/logo.svg';
+import facebook_logo from './icon/facebook.svg';
+import google_logo from './icon/google.png';
 import './App.css';
 
 const firebaseApp = firebase.initializeApp(firebaseConfig);
 
 class App extends Component {
+ 
+  state = {
+    user: {}
+  };
```

Comment:
why you inserted state into this component if it is not using it at all?

the HOC also handle the state management for the user instance, you shouldn't need this 😄